### PR TITLE
sno funnel registration flow tracking

### PR DESCRIPTION
**Pull request by crawter**

### CODE REVIEW

**Review by NikolaiYurchenko — APPROVED** [review]

File: layouts/partials/home/navbar.html [hunk 1]
```diff
@@ -16,8 +16,16 @@
...
           </div>
         </div>
     </nav>
+    <script>
+        const becomeHostButton = document.getElementById("become-a-host-btn");
+        if (becomeHostButton) {
+            becomeHostButton.addEventListener("click", function() {
+                analytics.identify(analytics.user().anonymousId(), {become_a_host_clicked: true});
+            })
+        }
```
Comment (crawter) [inline layouts/partials/home/navbar.html:30]: so here i track that users clicks "Become a Host" button

File: layouts/sign-up-node-operator-form/single.html [hunk 2]
```diff
@@ -58,6 +58,7 @@
             result[input.name] = input.value == "on";
         }
         errElm.setAttribute("class", "row invisible");
+        analytics.identify(analytics.user().anonymousId(), {sno_register_page_visited: true});
```
Comment (crawter) [inline layouts/sign-up-node-operator-form/single.html:61]: here i tracks that the same user has visited the first registration page

File: layouts/partials/sign-up-node-operator-email/form.html [hunk 3]
```diff
@@ -47,6 +47,7 @@ <h2 class="light-black">What’s your email?</h2>
             email = input.value
         }
         analytics.identify(email, {email: email, authToken: "unknown"});
+        analytics.identify(analytics.user().anonymousId(), {email: email, sno_register_email_page_visited: true});
```
Comment (crawter) [inline layouts/partials/sign-up-node-operator-email/form.html:50]: this call is pretty the same as a previous one, but i have to add a bit different data scheme so i have added a new call, not to break previous flow

**Review by grafael — APPROVED** [review]

**Review by mbouzi — APPROVED** [review]
👍

**Pull request merged**